api/spec/v1.yaml

```diff
@@ -2068,6 +2068,12 @@ components:
             The Oasis cryptographic hash of the transaction that created the smart contract.
              Can be omitted for contracts that were created by another contract, as opposed 
              to a direct `Create` call.
+        eth_creation_tx:
+          type: string
+          description: |
+            The Ethereum transaction hash of the transaction in `creation_tx`.
+            Encoded as a lowercase hex string.
```

Comment:
:grin: yeah I figured this is better than repeating the whole description of when it can be omitted, especially since the condition might change in the (far-ish) future if we implement contract execution tracing.